augur/application/schema/alembic/versions/18_schedule_any_old_facade_repositories_to.py

```diff
@@ -0,0 +1,77 @@
+"""Schedule any old facade repositories to be re-cloned
+
+Revision ID: 18
+Revises: 17
+Create Date: 2023-05-02 17:35:18.891913
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.sql import text
+import pathlib
+import shutil
+from augur.application.config import AugurConfig
+from augur.application.db.session import DatabaseSession
+import logging
+
+# revision identifiers, used by Alembic.
+revision = '18'
+down_revision = '17'
+branch_labels = None
+depends_on = None
+
+logger = logging.getLogger(__name__)
+
+
+def total_facade_reset():
+    conn = op.get_bind()
+
+    conn.execute(text(f"""
+
+    UPDATE augur_operations.collection_status
+    SET facade_status='Pending', facade_task_id=NULL, facade_weight=NULL,commit_sum=NULL,facade_data_last_collected=NULL;
+
+    UPDATE repo
+    SET repo_path=NULL,repo_name=NULL;
+
+    UPDATE augur_operations.collection_status
+    SET core_status='Pending', secondary_status='Pending', core_data_last_collected=NULL,secondary_data_last_collected=NULL
+    WHERE issue_pr_sum IS NULL;
+    """))
```

Comment:
Just to be safe I think we should also set the task ids to NULL

Reply:
Made the change